layouts/partials/blocks/carousel.html

```diff
@@ -0,0 +1,522 @@
+    }
+
+    function updateCarouselArrowStates() {
+      leftArrow.style.opacity = currentIndex === 0 ? "0.5" : "1";
+      leftArrow.style.pointerEvents = currentIndex === 0 ? "none" : "auto";
+      rightArrow.style.opacity =
+        currentIndex === carouselItems.length - 1 ? "0.5" : "1";
+      rightArrow.style.pointerEvents =
+        currentIndex === carouselItems.length - 1 ? "none" : "auto";
+    }
+
+    function navigateCarousel(direction) {
+      if (direction === "left" && currentIndex > 0) {
+        currentIndex--;
+      }
+      if (direction === "right" && currentIndex < carouselItems.length - 1) {
+        currentIndex++;
+      }
+      updateCarousel();
+    }
+
+    function scrollToActiveBox() {
+      const activeBox = document.querySelector(
+        `.impact-box[data-index="${currentIndex}"]`,
+      );
+      if (activeBox) {
+        const containerLeft = bottomBarContainer.scrollLeft;
+        const containerRight = containerLeft + bottomBarContainer.clientWidth;
+        const boxLeft = activeBox.offsetLeft;
+        const boxRight = boxLeft + activeBox.offsetWidth;
+
+        if (boxLeft < containerLeft) {
+          bottomBarContainer.scrollBy({
+            left: boxLeft - containerLeft,
+            behavior: "smooth",
+          });
+        } else if (boxRight > containerRight) {
+          bottomBarContainer.scrollBy({
+            left: boxRight - containerRight,
+            behavior: "smooth",
+          });
+        }
+      }
+    }
+
+    function handleBottomBoxClick(event) {
+      const clickedBox = event.target.closest(".impact-box");
+      if (clickedBox) {
+        const index = parseInt(clickedBox.dataset.index, 10);
+        if (!isNaN(index)) {
+          currentIndex = index;
+          updateCarousel();
+        }
+      }
+    }
+
+    leftArrow.addEventListener("click", () => navigateCarousel("left"));
+    rightArrow.addEventListener("click", () => navigateCarousel("right"));
+
+    impactBoxes.forEach((box) => {
+      box.addEventListener("click", handleBottomBoxClick);
+    });
+
+    const leftGradient = document.querySelector(".left-gradient-box");
+    const rightGradient = document.querySelector(".right-gradient-box");
+
+    function updateGradients() {
+      const scrollLeft = bottomBarContainer.scrollLeft;
+      const scrollWidth = bottomBarContainer.scrollWidth;
+      const clientWidth = bottomBarContainer.clientWidth;
+
+      if (scrollLeft > 0) {
+        leftGradient.style.display = "block";
+      } else {
+        leftGradient.style.display = "none";
+      }
+
+      if (scrollLeft + clientWidth < scrollWidth) {
+        rightGradient.style.display = "block";
+      } else {
+        rightGradient.style.display = "none";
+      }
+    }
+
+    bottomBarContainer.addEventListener("scroll", updateGradients);
+
+    window.addEventListener("resize", updateGradients);
+
+    updateCarousel();
+    updateGradients();
+  });
+</script>
+
+<style>
```

Comment:
This seems like it's probably sufficiently complex that it's not worth translating into Tailwind, but we can move it into assets/css/carousel.css and link it from there just to simplify the page a little.

```
{{ $css := resources.Get "css/carousel.css" | minify | fingerprint }}
<link rel="stylesheet" href="{{ $css.RelPermalink }}" />
```